cmd/gke-gcloud-auth-plugin/cred/token_provider.go

```diff
@@ -9,4 +9,6 @@ type tokenProvider interface {
 	token() (token string, expiry *time.Time, err error)
 	// useCache returns whether or not tokens from this providers should be cached
 	useCache() bool
+	// context returns the token provider context for the token generated
```

Comment:
This comment doesn't seem descriptive, given "context" here seems to refer more to a provider name than a specific request's context. Maybe providerName might be more descriptive, or something else more explicit?